onnxscript/onnx_opset/__init__.py

```diff
@@ -0,0 +1,130 @@
+# --------------------------------------------------------------------------
+# ⚠️ WARNING - AUTO-GENERATED CODE - DO NOT EDIT ⚠️
+# ⚙️ Generated by: setup.py opgen
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from onnx.defs import onnx_opset_version
+from onnx.onnx_cpp2py_export.defs import schema_version_map  # type: ignore
+
+from ..values import Opset
+from ._impl.opset1 import Opset1 as __Opset1
+from ._impl.opset2 import Opset2 as __Opset2
+from ._impl.opset3 import Opset3 as __Opset3
+from ._impl.opset4 import Opset4 as __Opset4
+from ._impl.opset5 import Opset5 as __Opset5
+from ._impl.opset6 import Opset6 as __Opset6
+from ._impl.opset7 import Opset7 as __Opset7
+from ._impl.opset8 import Opset8 as __Opset8
+from ._impl.opset9 import Opset9 as __Opset9
+from ._impl.opset10 import Opset10 as __Opset10
+from ._impl.opset11 import Opset11 as __Opset11
+from ._impl.opset12 import Opset12 as __Opset12
+from ._impl.opset13 import Opset13 as __Opset13
+from ._impl.opset14 import Opset14 as __Opset14
+from ._impl.opset15 import Opset15 as __Opset15
+from ._impl.opset16 import Opset16 as __Opset16
+from ._impl.opset17 import Opset17 as __Opset17
+from ._impl.opset_ai_onnx_ml1 import Opset_ai_onnx_ml1 as __Opset_ai_onnx_ml1
+from ._impl.opset_ai_onnx_ml2 import Opset_ai_onnx_ml2 as __Opset_ai_onnx_ml2
+from ._impl.opset_ai_onnx_ml3 import Opset_ai_onnx_ml3 as __Opset_ai_onnx_ml3
+from ._impl.opset_ai_onnx_preview_training1 import (
+    Opset_ai_onnx_preview_training1 as __Opset_ai_onnx_preview_training1,
+)
+
+all_opsets: list[Opset] = []
```

Comment:
List?

Reply:
Resolving this to [keep discussion below](https://github.com/microsoft/onnx-script/pull/162#discussion_r1016849642).